internal/controllers/device_data_controller.go

```diff
@@ -344,31 +344,30 @@ func (udc *UserDevicesController) QueryDeviceErrorCodes(c *fiber.Ctx) error {
 // @Tags        user-devices
 // @Success     200 {object} controllers.GetUserDevicesErrorCodeQueriesResponse
 // @Security    BearerAuth
-// @Router      /user/devices/error-codes [get]
+// @Router      /user/devices/{userDeviceID}/error-codes [get]
 func (udc *UserDevicesController) GetUserDevicesErrorCodeQueries(c *fiber.Ctx) error {
```

Comment:
Oh maybe we should change the function name too—`GetUserDeviceErrorCodeQueries`?